stories/Atom/Typography/Links/link.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import './link.scss';
 
 export const Link = ({ label }) => (
-  <a href="#">
+  <a href="#" role="link" alt={label} title={label}>
```

Comment:
@harishkumar-srijan i don't think alt is the attribute of anchor tag and do we need the role='link' attribute on anchor

Reply:
done